app/com/linkedin/drelephant/tuning/AutoTuningAPIHelper.java

```diff
@@ -18,6 +18,8 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.linkedin.drelephant.ElephantContext;
+import com.linkedin.drelephant.tuning.obt.AutoTuningOptimizeManager;
+import com.linkedin.drelephant.tuning.obt.OptimizationAlgoFactory;
```

Comment:
Don't we have to change getCurrentRunParameters method? Now the tuning type input would come from UI. Also TuningJobDefinition is set from this method